tests/org.eclipse.fordiac.ide.test.ui/src/org/eclipse/fordiac/ide/test/ui/PropertySheetHelper.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2009-2019 THALES GLOBAL SERVICES
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *      Obeo - Initial API and implementation
+ *      Prashantkumar Khatri -  Integrated selectPropertyTabItem and
+ *      						createEvent from
+ *      						"org.eclipse.sirius.tests.swtbot.support.api.editor.SWTBotSiriusHelper"
+ */
+package org.eclipse.fordiac.ide.test.ui;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Event;
+import org.eclipse.swt.widgets.Widget;
+import org.eclipse.swtbot.eclipse.finder.SWTWorkbenchBot;
+import org.eclipse.swtbot.eclipse.finder.matchers.WidgetMatcherFactory;
+import org.eclipse.swtbot.swt.finder.SWTBot;
+import org.eclipse.swtbot.swt.finder.finders.UIThreadRunnable;
+import org.eclipse.swtbot.swt.finder.results.BoolResult;
+import org.eclipse.swtbot.swt.finder.utils.SWTUtils;
+import org.eclipse.ui.internal.views.properties.tabbed.view.TabbedPropertyList;
+import org.hamcrest.Matcher;
+import org.hamcrest.Matchers;
+
+public class PropertySheetHelper {
+
+	/**
+	 * SWTWorkbenchBot
```

Comment:
as discussed earlier, please keep the comment that you also had in the previous version, showing the exact location of this method in SWTBotHelper from Sirius